examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -55,49 +55,37 @@ async def _main():
         try:
             client = grpc_nidaqmx.NiDAQmxStub(channel)
 
-            async def raise_if_error(response):
-                """Raise an exception if an error was returned."""
-                if response.status:
-                    response = await client.GetErrorString(
-                        nidaqmx_types.GetErrorStringRequest(error_code=response.status)
+            def check_for_warning(response):
+                """Print to console if the status indicates a warning."""
+                if response.status > 0:
+                    warning_message = client.GetErrorStringResponse(
+                        nidaqmx_types.ErrorMessageRequest(error_code=response.status)
                     )
-                    raise Exception(f"Error: {response.error_string}")
+                    sys.stderr.write(f"{warning_message.error_message}\nWarning status: {response.status}\n")
 
-            async def raise_if_error_async(awaitable_call):
-                """Await response, then raise an exception if an error was returned."""
-                response = await awaitable_call
-                await raise_if_error(response)
-                return response
-
-            create_response: nidaqmx_types.CreateTaskResponse = await raise_if_error_async(
-                client.CreateTask(nidaqmx_types.CreateTaskRequest())
-            )
+            create_response: nidaqmx_types.CreateTaskResponse = await client.CreateTask(nidaqmx_types.CreateTaskRequest())
```

Comment:
Is it necessary to declare the `CreateTaskResponse` type here? It's not done anywhere else that I can see.

Reply:
Potentially not, but I'm also going to leave this untouched as it's out of scope for the current PR (which I'm hoping to merge today).